doc/src/refine.rst

```diff
@@ -653,6 +783,13 @@ where ``Equiv`` is defined by
 
 :index:`V/eq/uain`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+
+  H >> (vin #r #m0 #n0) = (vin #r #m1 #n1) in (V #r #a #b #e)
+  | H, #r=0 >> #m0 = #m1 in #a
+  | H >> #n0 = #n1 in #b
+  | H, #r=0 >> ($ (! f #e) #m0) = #n0 in #b
```

Comment:
This should be `! proj1` instead of `! f`

Reply:
fixed